.eslintignore

```diff
@@ -0,0 +1 @@
+test/bigint-buffer.js
```

Comment:
This should be removed since only place you are using `BigInt` is [3 lines here](https://github.com/metarhia/metacom/pull/24/files/3988252f26450e6b5468aad444e05496f8937583..9825f7298114e24a658da17e05f6c86c6b82a572#diff-e7aef560573fc1070565760c0c102076R11)

Reply:
If eslint don't ignore file, it will perform `Parsing error` which can't be fixed with disabling linter inline or for block of code